WebRoot/skins/_base/base4/skin.css

```diff
@@ -1136,6 +1136,10 @@ svg.progress-circle.tick-inside {
     padding-left:13px;
 }
 
+.ZmCalListViewSearchBar .DwtText {
+    font-weight: bold;
```

Comment:
can we move this to zm.css and create a property so skin developers can override it?

Reply:
Just a text at the calendar list view search bar. We are styling it on this theme (just making all the text bold on the toolbar) . Don't think we would want to expose this particular style rule. 
Let me know your inputs.

Reply:
ok